nwb_conversion_tools/utils/spike_interface.py

```diff
@@ -460,7 +460,8 @@ def add_electrical_series(
     use_times: bool = False,
     write_as: str = 'raw',
     es_key: str = None,
-    write_scaled: bool = False
+    write_scaled: bool = False,
+    use_compression: bool = True
```

Comment:
better to have compression='gzip', mirroring [h5py syntax](https://docs.h5py.org/en/stable/high/dataset.html#filter-pipeline)